gem/migrations/0014_convert_recomended_articles.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_recomended_articles(main_article, article_list):
+    '''
+    Creates recommended article objects from article_list
+    and _prepends_ to existing recommended articles.
+    '''
+    from molo.core.models import ArticlePageRecommendedSections
+
+    existing_recommended_articles = [
+        ra.recommended_article.specific
+        for ra in main_article.recommended_articles.all()]
+    ArticlePageRecommendedSections.objects.filter(page=main_article).delete()
+
+    for hyperlinked_article in article_list:
+        ArticlePageRecommendedSections(
+            page=main_article,
+            recommended_article=hyperlinked_article).save()
+
+    # re-create existing recommended articles
+    for article in existing_recommended_articles:
+        if article not in article_list:
+            ArticlePageRecommendedSections(
+                page=main_article,
+                recommended_article=article).save()
+
+
+def convert_articles(apps, schema_editor):
+    '''
+    Derived from https://github.com/wagtail/wagtail/issues/2110
+    '''
+    from molo.core.models import ArticlePage
+    from wagtail.wagtailcore.blocks import StreamValue
+
+    articles = ArticlePage.objects.all()
+
+    for article in articles:
+        stream_data = []
+        linked_articles = []
+        for block in article.body.stream_data:
+            if block['type'] == 'page':
+                try:
+                    linked_articles.append(ArticlePage.objects.get(
+                                    id=block['value']))
+                except:
```

Comment:
What error are you expecting here? It's better to be explicit rather than a blanket catch all.